### PR TITLE
Fix: Xdai -> Gnosis

### PR DESCRIPTION
**Pull request by moconnell**
- [x] replace ../deployments/xdai.json with ..gnosis.json

### CODE REVIEW

**Review by psparacino — CHANGES_REQUESTED** [review]
Also need to change the network value in deployments/gnosis, looks like it's still currently xdai. Also cam change in hardhat config. Every reference to xdai (except potentially block explorer urls can be changed to gnosis I believe

**Comment by moconnell** [comment]
> Also need to change the network value in deployments/gnosis, looks like it's still currently xdai. Also cam change in hardhat config. Every reference to xdai (except potentially block explorer urls can be changed to gnosis I believe

ok, have fixed all the other locations I could find, apart from any references to the subgraph `psparacino/v1-xdai-smart-invoices`

File: packages/contracts/tasks/verify-blockscout.js [hunk 1]
```diff
@@ -13,12 +13,12 @@ const BLOCKSCOUT_CHAIN_IDS = [77, 100];
 
 const API_URLS = {
   77: "https://blockscout.com/poa/sokol/api",
-  100: "https://blockscout.com/xdai/mainnet/api",
+  100: "https://gnosis.blockscout.com/api",
```
Comment (psparacino) [inline packages/contracts/tasks/verify-blockscout.js:16]: I'm guessing our hardhat version is going to be too old to verify with this url. But maybe that's ok and we can just verify manually. Good incentive to also update the hardhat version, or even Foundry, but those are big and bigger lifts respectively.

File: packages/dapp/constants/index.js [hunk 2]
```diff
@@ -78,7 +78,7 @@ export const explorerUrls = {
   4: 'https://rinkeby.etherscan.io',
   5: 'https://goerli.etherscan.io',
   42: 'https://kovan.etherscan.io',
-  100: 'https://blockscout.com/poa/xdai',
+  100: 'https://gnosis.blockscout.com',
```
Comment (psparacino) [inline packages/dapp/constants/index.js:81]: same here

**Review by psparacino — APPROVED** [review]
The comments don't preclude a merge in my opinion to test on develop.  More that we very likely need to update the hardhat version to be able to verify

**Pull request merged**